openstudiocore/src/model/ThermalZone.cpp

```diff
@@ -1998,18 +1998,13 @@ namespace detail {
   {
     boost::optional<AirLoopHVACSupplyPlenum> result;
 
-    std::vector<AirLoopHVACSupplyPlenum> plenums = model().getModelObjects<AirLoopHVACSupplyPlenum>();
+    std::vector<AirLoopHVACSupplyPlenum> plenums = getObject<ModelObject>().getModelObjectSources<AirLoopHVACSupplyPlenum>();
```

Comment:
Never been a fan of seeing the method getModelObjectSources in use. If the performance gain can be achieved using getConcreteModelObjects but leaving the method the same, I'm more in favor of that.